game/randomGen.cc

```diff
@@ -0,0 +1,13 @@
+#include "randomGen.h"
+#include <iostream>
+
+using namespace std;
+
+RandomGenerator::RandomGenerator(const unsigned int min_, const unsigned int max_) : min{min_}, max{max_} {
+  random_generator = mt19937 {rd()};
```

Comment:
`random_generator` is a static field, so it shouldn't be initialized in the constructor